modVM/src/org/aion/vm/ExecutionContext.java

```diff
@@ -120,6 +121,8 @@ public ExecutionContext(
         this.blockNrgLimit = blockNrgLimit;
         this.blockDifficulty = blockDifficulty;
         this.txHash = txHash;
+        this.originalTxHash = txHash;
```

Comment:
The `txHash` is actually the original transaction hash. Suggest removing one of the them.

Reply:
@yulongaion not really, when the VM executing the internal tx, the txHash will be modified to the hash of the internal tx.